components/cache/adapters/apcu_adapter.rst

```diff
@@ -17,7 +17,7 @@ memory, a component appreciably faster than many others, such as the filesystem.
     this adapter.
 
 The ApcuAdapter can optionally be provided a namespace, default cache lifetime,
-and cache items version string as constructor arguments::
+cache items version string and marshaller as constructor arguments::
```

Comment:
We probably should add reference to marshallers docs: https://symfony.com/doc/current/components/cache.html#marshalling-serializing-data
```
:ref:`marshaller <marshalling-serializing-data>`
```

`components/cache.rst` must also be updated from

```
Marshalling (Serializing) Data
------------------------------
```

to

```
.. _marshalling-serializing-data:

Marshalling (Serializing) Data
------------------------------
```